### PR TITLE
update v1 and v2 to its latest minor/patch version

### LINKED ISSUE #24: update chain-maind v1 to v1.2.2 and v2 to v2.1.2
- v1.2.2 supports Arm64 (Linux and macOS)
- v2.1.2 contains a fix for the non-deterministic app hash mismatch
issues after node restarts or state sync

### PR DESCRIPTION
**Pull request by allthatjazzleo**
close #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +26,13 @@ $ brew upgrade chain-maind
 # Install previous version before Canis Major network upgrade 
 
 ```bash
-$ brew install chain-maind@1.2.1
+$ brew install chain-maind@1.2.2
 ```
 
 # Install previous version before DRACO II Major network upgrade
```
Comment (lezzokafka) [inline README.md]:
```suggestion
# Install previous version before DRACO II Major network upgrade (block height before `3,526,800`)
```

File: README.md [hunk 2]
```diff
@@ -26,13 +26,13 @@ $ brew upgrade chain-maind
 # Install previous version before Canis Major network upgrade
```
Comment (lezzokafka) [inline README.md]:
```suggestion
# Install previous version before Canis Major network upgrade (block height before `922,363`)
```

**Review by lezzokafka — COMMENTED** [review]
Left some quick suggestions, otherwise looks great :D

**Comment by allthatjazzleo** [comment]
added upgrade block height

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**